components/camel-smpp/src/main/java/org/apache/camel/component/smpp/SmppConfiguration.java

```diff
@@ -120,6 +120,8 @@ public class SmppConfiguration implements Cloneable {
     private Integer pduProcessorQueueCapacity = 100;
     @UriParam(label = "advanced", defaultValue = "false")
     private boolean singleDLR;
+    @UriParam(label = "advanced", enums = "0x00,0x33,0x34,0x50", defaultValue = "0x34")
+    private String interfaceVersion = "0x34";
```

Comment:
Maybe just use a plain string to set the value in better human understandable as just a string with 3.3, 3.4, 5.0 etc. And then internally convert that to what jsnmp uses.

Reply:
The SMPP protocol uses these codes (e.g. se https://smpp.org/SMPP_v5.pdf - chapter 4.7.13) in its specification. Are you sure I should remove them? (Code is simpler/easier to understand like this)

Reply:
I have introduced a new scheme using "legacy", "3.3", "3.4", "5.0"